backend/user/management/commands/send_shadow_notifs.py

```diff
@@ -0,0 +1,41 @@
+import json
+
+from django.core.management.base import BaseCommand
+
+from user.models import NotificationToken
+from user.notifications import send_shadow_push_notif_batch
+
+
+class Command(BaseCommand):
+    help = "Sends shadow notifs to all or targeted users."
+
+    def handle(self, *args, **kwargs):
+        send_to_all = input("Send to all (yes/no)? ").lower() == "yes"
+        # get list of targeted users if not to everyone
+        if not send_to_all:
+            usernames = input("Target users (comma-delimitted): ").split(",")
```

Comment:
for lines 16 and 18, do you mind also adding an example of the input we are looking for? just want to add this cause i don't want others to continuously ask us what the inputs for this looks like haha